npm/builder.js

```diff
@@ -487,6 +490,17 @@ Builder.prototype.compile = function() {
   fs.createReadStream(asciidoctorCSSFile).pipe(fs.createWriteStream('build/asciidoctor.css'));
 };
 
+Builder.prototype.replaceUnsupportedFeatures = function() {
+  log.title('Replace unsupported features');
+  var path = 'build/asciidoctor-core.js';
+  var data = fs.readFileSync(path, 'utf8');
+  log.debug('Replace sub! with sub');
+  data = data.replace(/\['\$sub!'\]/gi, '[\'$sub\']');
```

Comment:
I tested this replacement and it does exactly what we want and doesn't seem to cause any issues.